spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/config/StandardizationConfigParser.scala

```diff
@@ -137,64 +148,66 @@ object StandardizationConfigParser {
         .action((value, config) => config.withFixedWidthTreatEmptyValuesAsNulls(Some(value)))
         .text("For FixedWidth file format. Treats empty values as null. Default is false"),
 
-      opt[String]("null-value").optional()
-        .action((value, config) => config.withFixedWidthNullValue(Some(value)))
-        .text("""For FixedWidth file format. Sets the representation of a null value. Defaults is ""."""),
-
       checkConfig(checkConfigX(_, builder))
     )
   }
   //scalastyle:on method.length
 
   private val formatsSupportingCharset = List("xml", "csv", "json", "cobol", "fixed-width")
 
-  private def typicalError(field: String, format: String): String = {
-    s"The $field option is supported only for $format format"
+  private def unsupportedOptionError(option: String, format: String): String = {
+    unsupportedOptionError(option, Seq(format))
+  }
+
+  private def unsupportedOptionError(option: String, formats: Seq[String]): String = {
+    def sprintf(frmt: String, s: String = ""): String = s"The $option option is supported only for $frmt format$s"
+
+    formats match {
+      case Seq()       => ""
+      case Seq(format) => sprintf(format)
+      case _           =>
+        val revertedFormats = formats.reverse
+        val frmt = revertedFormats.reverse.mkString(", ") + " and " + revertedFormats.head
```

Comment:
Doubles last of the formats. 
`The --null-value option is supported only for CSV, FixedWidth and FixedWidth formats`

Reply:
Thanks. Will fix. This is why I wanted to extract this - to also make it testable.